service/internal/infrastructure/infrastructure.go

```diff
@@ -4,6 +4,8 @@ const (
 	AWSClusterKind         = "AWSCluster"
 	AWSClusterKindProvider = "capa"
 
+	AWSManagedClusterKind = "AWSManagedCluster"
```

Comment:
Is it worth adding the equivalent kinds for `AzureManagedCluster` and `GCPManagedCluster` alongside this?

Reply:
It surely is, I am though not sure the values both should get. For `AWSManagedCluster` I concluded from what we have on `snail` it should be mapped to `capa` and get the same values as regular `AWSCluster`, not sure it is the same for others, but I would bet on that